content/blog/2020/08/2020-08-12-windows-installers-upgrade.adoc

```diff
@@ -0,0 +1,102 @@
+
+ 1. The new installer will only support 64-bit systems going forward. This is the vast majority of Windows systems these days, so this will help more users install Jenkins using the installer package.
+ 2. The user is now able to enter user information for the service and select the port that Jenkins will use and verify availability of the port.
+ 3. The installer no longer bundles a JRE, but will search for a compatible JRE on the system. If the user wants to use a different JRE, it may be specified during install.
+ 4. The installer has support for running with a Java 11 JRE, including the components listed on the link:/blog/2018/12/14/java11-preview-availability/[Java 11 Preview Page].
+ 5. The JENKINS_HOME directory is placed in the LocalAppData directory for the user that the service will run as, this aligns with modern Windows file system layouts.
+ 6. The installer has been updated with branding to make it look nicer and provide a better user experience. 
+ 
+## Screenshots
+
+You may see below the sequence of screenshots for the new installer:
+
+### Step 1
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_1.png[Installer Startup, role=center]
+
+We can see now the Jenkins logo as a prominent part of the unstaller UI.
+
+### Step 2
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_2.png[Installation Directory, role=center]
+
+Now the Jenkins logo and name are in the header during entire process of installation.
+
+### Step 3
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_3.png[Account Selection, role=center]
+
+Now the installer allows both specifying and testing your credentials by validating that the account has LogonAsService rights.
+
+### Step 4
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_4.png[Port Selection, role=center]
+
+Now the installer also allows specifying the port that Jenkins should run on and will not continue until a valid port is entered and tested.
+
+### Step 5
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_5.png[JRE Selection, role=center]
+
+Now instead of bundling a JRE, the installer searches for a compatible JRE on the system (in the current search no JRE was installed). In case you would like to use a different JRE from the one found by the installer, you can browse and specify it. Only JRE 8 and JRE 11 runtimes are supported. In case the selected JRE is found to be version 11 the installer will
+automatically add the necessary arguments and additional jar files for running under Java 11.
+
+### Step 6
+
+image:/images/post-images/2020-08-07-win-installers-upgrade/new_installer_6.png[Install It, role=center]
+
+All of the items that users can enter in the installer should be overridable on the command line for automated deployment as well. The full list of properties that can be overridden will be available soon.

```

Comment:
That's a nice capability that I didn't realize was available.  @slide , is there a list of command line arguments that should be described?
```suggestion
All of the items that users can enter in the installer should be overridable on the command line for automated deployment as well. The full list of properties that can be overridden will be available soon.

```

Reply:
It would be nice to describe it. I can try to pull something together.